edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/planner/CreateUpdateCommandMaker.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Core.Planner
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Agent.Core.Commands;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class CreateUpdateCommandEntity
```

Comment:
suggesting new files for each class/interface

Reply:
Sounds good. But waiting to make the change until we align on command factory behavior.